### DISCUSSION TITLE
error in character count :(

### DISCUSSION THREAD

**Discussion by cypriano-design** [cssbattle/forum#183] [Bugs reports]
Hi guys! I finished challenge #89 and, even though I'm not the master of css, I was playing at reducing the code and managed to get down to 490 characters! However, when I submit to see the result...it says I used 645 :(

What can it be?

Print: [https://imgur.com/a/haTT7o5](https://imgur.com/a/haTT7o5)

**Comment by Beowolve** [comment]
You only reached 99.7% and that is why you still see your current best highest score with 645 chars. It is very important to make a 100% solution to reach the highest score.

**Reply by cypriano-design** [reply]
ahhhhh got it! It's based on the number of characters in my best result. Thanks!

Unfortunately, my knowledge doesn't allow me to improve much more than 0.01% 😢 It was the first time I made a triangle in CSS 😅 And I can't get the edge rounded enough to make the fit. But I will search ☺️ Thanks!

**Reply by chinchang** [reply]
To add to what @Beowolve said, solutions are ranked by score - not just match percentage and not just low code count, but score. And score is made of match percentage and character count both. And a better match solution always has higher score than lower match solution.
https://cssbattle.dev/faqs#how-are-scores-calculated
